frontend/src/components/Input/Input.tsx

```diff
@@ -37,6 +37,10 @@ export interface InputProps extends ChakraInputProps {
    * Whether there's an input right element. Used to provide additional padding
    */
   hasInputRightElement?: boolean
+  /**
+   * Whether the input is in a high contrast state. (e.g For MRF prefills)
```

Comment:
this makes sense as it includes why this was included - for MRF prefills 

prefer to make the comment more generic eg, 
- `isHighContrast is used to allow values to be read easily when state is disabled`